src/frontend/screens/MapScreen/CurrentTrack/UserTooltipMarker.tsx

```diff
@@ -3,21 +3,21 @@ import {StyleSheet, Text, View} from 'react-native';
 import React from 'react';
 import {usePersistedTrack} from '../../../hooks/persistedState/usePersistedTrack';
 import {useTrackTimerContext} from '../../../contexts/TrackTimerContext';
-import {useSharedLocationContext} from '../../../contexts/SharedLocationContext';
+import {useLocation} from '../../../hooks/useLocation';
+import {TRACKING_DISTANCE_INTERVAL} from '../../../constants';
 
 export const UserTooltipMarker = () => {
   const {timer} = useTrackTimerContext();
-  const {locationState} = useSharedLocationContext();
+  const {location} = useLocation({
+    maxDistanceInterval: TRACKING_DISTANCE_INTERVAL,
+  });
```

Comment:
> I _think_ this needs to also be updated, otherwise the placement of the track timer tooltip can be misaligned:
> 
> https://github.com/digidem/comapeo-mobile/blob/312c140674979caae5d0c6931972f5aadf53d510/src/frontend/contexts/SharedLocationContext.tsx#L23
> 

Unfortunately I couldn't just update the location context as that would effect how often the map gets optimized. So i am just using this hook instead.

That being said, I am proposing a new location architecture that will fix all of this (this PR is why I am proposing that) so they can all used a shared location context